sudoku-mw/src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -25,7 +26,7 @@
     <![endif]-->
 </head>
 
-<body>
+<body onload="show()">
```

Comment:
this is old style, use jquery standard approach `(document).ready(function() {...`